cmd/image.go

```diff
@@ -96,40 +93,28 @@ kubeaudit image -i gcr.io/google_containers/echoserver:1.7`,
 		if rootConfig.json {
 			log.SetFormatter(&log.JSONFormatter{})
 		}
+		var resources []Items
```

Comment:
This whole thing could be wrapped in single function which would return a list of Items i.e. check if manifest is nil or not. Thus reducing all the redundant code.

Reply:
Yes, that's the next step :)